cellpack/tests/packing-configs/test_gradient_data_image.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "test_gradient_data_image",
+    "format": "simularium",
+    "inner_grid_method": "trimesh",
+    "live_packing": false,
+    "ordered_packing": false,
+    "out": "out/",
+    "overwrite_place_method": true,
+    "place_method": "spheresSST",
+    "save_analyze_result": true,
+    "show_grid_plot": true,
+    "number_of_packings": 1,
+    "spacing": 0.25,
+    "use_periodicity": false,
+    "show_sphere_trees": false,
+    "load_from_grid_file": true,
+    "save_converted_recipe": true,
+    "image_export_options": {
+        "hollow": false,
+        "voxel_size": [1,1,1],
+        "projection_axis": "z"
+    },
+    "save_gradient_data_as_image": true
```

Comment:
if this is a new config setting it should also have a default setting in the config loader. 
I also think, for clarity, any settings here that are defaults should be removed. The configs are ballooning a bit

Reply:
Removed the default entries from the config file and added the new setting to `config_loader.py`